src/int.js

```diff
@@ -121,18 +128,22 @@ Sk.builtin.int_ = function (x, base) {
     return this;
 };
 
+Sk.abstr.setUpInheritance("int", Sk.builtin.int_, Sk.builtin.numtype);
+
 Sk.builtin.int_.co_varnames = [ "base" ];
 Sk.builtin.int_.co_numargs = 2;
-Sk.builtin.int_.$defaults = [ new Sk.builtin.int_(10) ];
```

Comment:
Good to know! I'll move those too. It's a bit easier to initialize all the constants in one place in case there are any internal dependencies in their constructors.